api/task/target_rank.go

```diff
@@ -16,63 +16,55 @@
 package task
 
 import (
-	"fmt"
-	"path/filepath"
-	"strconv"
-
-	log "github.com/unchartedsoftware/plog"
-
-	"github.com/pkg/errors"
 	"github.com/uncharted-distil/distil-compute/metadata"
 	"github.com/uncharted-distil/distil-compute/model"
-	"github.com/uncharted-distil/distil-compute/primitive/compute/description"
-	"github.com/uncharted-distil/distil-compute/primitive/compute/result"
-	"github.com/uncharted-distil/distil/api/env"
 )
 
 // TargetRank will rank the dataset relative to a target variable using
 // a primitive.
 func TargetRank(dataset string, target string, features []*model.Variable, source metadata.DatasetSource) (map[string]float64, error) {
-	// create & submit the solution request
-	pip, err := description.CreateTargetRankingPipeline("roger", "", target, features)
-	if err != nil {
-		return nil, errors.Wrap(err, "unable to create ranking pipeline")
-	}
+	// // create & submit the solution request
+	// pip, err := description.CreateTargetRankingPipeline("roger", "", target, features)
+	// if err != nil {
+	// 	return nil, errors.Wrap(err, "unable to create ranking pipeline")
+	// }
+
+	return map[string]float64{}, nil
```

Comment:
Did you mean to leave this whole thing commented out?

Reply:
No, I will re-enable it and re-test.